docker-compose.yaml

```diff
@@ -11,16 +9,19 @@ services:
         container_name: ankerctl
         restart: unless-stopped
         build: .
+
+        # host-mode networking is required for pppp communication with the
+        # printer, since it is an asymmetrical udp protocol.
+        network_mode: host
+
+        # bind to localhost by default
         environment:
+            - FLASK_HOST=127.0.0.1
             - FLASK_PORT=4470
-            - FLASK_HOST=0.0.0.0
         volumes:
             - ankerctl_vol:/root/.config/ankerctl
         ports:
             - 127.0.0.1:4470:4470
-        working_dir: /app
```

Comment:
Just a comment: 
currently `working_dir` is not defined in either the Dockerfile or the docker-compose. This isn't a blocker, but might cause some confusion down the line if the entrypoint or other script gets added. 

While definitely not a "best practice", I consider explicitly defining the working_dir to be a good idea.

Reply:
We have this in the non-build env in `Dockerfile`:

```
# Set the working directory to /app
WORKDIR /app
```

Does that cover it?

Reply:
Oh yes, thats great.